packages/amplify-nodejs-function-runtime-provider/src/__tests__/utils/__snapshots__/execute.test.ts.snap

```diff
@@ -0,0 +1,11 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`execute async handler should fail when throwing an error 1`] = `Object {}`;
```

Comment:
@edwardfoyle I found a possible issue here.
As you can see the result is `Object {}`. I discovered that this is due to the `_.assign` [here](https://github.com/aws-amplify/amplify-cli/blob/master/packages/amplify-nodejs-function-runtime-provider/src/utils/execute.ts#L40)

I guess the goal is to clone the object here?

Reply:
To be honest, I'm not sure what the intent of assigning to an object there was, but should be fine to leave it